bccsp/schemes/dlog/crypto/idemix_test.go

```diff
@@ -43,6 +43,20 @@ func TestIdemixGurvy254(t *testing.T) {
 	testIdemix(t, curve, tr)
 }
 
+func TestIdemixBLS12_381(t *testing.T) {
```

Comment:
shall we compact these tests with different curve in a single test that test in parallel all the curves supported by mathlib?

Reply:
good idea - done